src/routes/DesignSystemApp.svelte

```diff
@@ -107,9 +109,45 @@
     </ONSCollapsible>
   </DesignSystemPanel>
 
-  <DesignSystemPanel title="Input" code={`<ONSTextField id="text-field">Text field</ONSTextField>`}>
-    <ONSTextField id="text-field">Text field</ONSTextField>
+  <DesignSystemPanel
```

Comment:
I like the two separate examples 👍

Reply:
Because now we have a placeholder I've updated the _Input_ (title-Input)  component to display the placeholder.